tests/fixtures/browser.ts

```diff
@@ -245,22 +247,29 @@ export const createMockBrowserState = () => {
         trackConfigNames: createTrackConfigNames()
       },
       trackPanel: {
+        [fakeGenomeId]: {
+          selectedTrackPanelTab: TrackSet.GENOMIC,
+          isTrackPanelOpened: true,
+          highlightedTrackId: '',
+          collapsedTrackIds: []
+        }
+      },
+      browserSidebarModal: {
+        [fakeGenomeId]: {
+          browserSidebarModalView: BrowserSidebarModalView.SEARCH
```

Comment:
Do we need the sidebar search to be shown by default for testing?

Reply:
Should this be `null` instead then?

Reply:
✅